ice.c

```diff
@@ -122,7 +125,12 @@ void janus_ice_force_rtcpmux(gboolean forced) {
 			JANUS_LOG(LOG_WARN, "Error creating RTCP component blackhole socket, using port %d instead\n", janus_force_rtcpmux_blackhole_port);
 			return;
 		}
+#ifdef _WIN32
+		u_long argp = 1;
+		ioctlsocket(blackhole, FIONBIO, &argp);
+#else
```

Comment:
This is one of the things that frighten me, I'll most definitely break those anytime I add something sockety myself :wink: